### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by eanylin**
Updated README with Mission, Roadmap, Getting
Started and Bugs

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,5 +1,49 @@
 ## Shipyard ##
 
-Shipyard adopts the Falcon web framework and uses Apache Airflow as backend engine to programmatically 
-author, schedule and monitor workflows
+Shipyard adopts the Falcon web framework and uses Apache Airflow as backend engine to programmatically
```
Comment (intlabs) [inline docs/README.md]: should this read: `as the backend engine`

**Review by intlabs — APPROVED** [review]
Minor nit - but looks great otherwise :)

File: docs/README.md (hunk 1 above)
Comment (eanylin) [reply docs/README.md]: Agree. Updated.

**Review by mark-burnett — APPROVED** [review]
LGTM.  Thanks for the expanded docs @eanylin

**Pull request merged**